packages/src/lib/lateral-menu/lateral-menu.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface Section {
+  title?: string;
+  path?: string;
+  subsections?: Section[];
+  opened?: boolean;
+}
```

Comment:
Je trouve ça anormal d'avoir une interface avec tous ses propriétés optionnelles. Souvent c'est un signe qu'on pourrait l'améliorer ou qu'on a oublié des éléments requis.

On pourrait p-t décortiquer en deux cette interface, l'item et celui de section qui a des enfants? Ex;
```suggestion
export type AnySection = Section | SectionItem;

export interface Section extends SectionBase {
  subsections: AnySection[];
  opened?: boolean;
}

export interface SectionItem extends SectionBase {
  path: string;
}

export interface SectionBase {
  title: string;
}

```

Reply:
J'ai ajusté un peu comme les sections de `block-link`